specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json

```diff
@@ -11636,15 +11645,15 @@
         },
         "usesTaskDependencies": {
           "type": "boolean",
-          "title": "The flag that determines if this job will use tasks with dependencies."
+          "title": "Specifies whether tasks in the job can define dependencies on each other. The default is false."
```

Comment:
>Specifies whether [](start = 20, length = 17)

Is 'specifies' safe here?  I thought we preferred phrases that could be prefixed with "Gets or sets", e.g. "Whether...", but perhaps that's no longer needed?